perf/newrawkey.c

```diff
@@ -70,25 +77,45 @@ int main(int argc, char *argv[])
         return EXIT_FAILURE;
     }
 
+    times = OPENSSL_malloc(sizeof(OSSL_TIME) * threadcount);
+    if (times == NULL) {
+        printf("Failed to create times array\n");
+        return EXIT_FAILURE;
+    }
+
     if (!perflib_run_multi_thread_test(do_newrawkey, threadcount, &duration)) {
         printf("Failed to run the test\n");
-        return EXIT_FAILURE;
+        goto out;
     }
 
     if (err) {
         printf("Error during test\n");
-        return EXIT_FAILURE;
+        goto out;
     }
 
-    us = ossl_time2us(duration);
-
-    avcalltime = (double)us / NUM_CALL_BLOCKS_PER_RUN;
+    us = times[0];
+    for (i = 1; i < threadcount; i++)
+        us = ossl_time_add(us, times[i]);
+    us = ossl_time_divide(us, NUM_CALLS_PER_TEST);
```

Comment:
Shouldn't we convert `us` to double before this divide?

Reply:
I do that below when converting to an average number of microseconds.  I suppose we could do it here in all the tests instead.  i.e.:
```
av = ((double)(ossl_time_to_ticks(us)/(double)(NUM_CALLS_PER_TEST)) / (double)(OSSL_TIME_US)
```
Do you think that would make more sense?

Reply:
This also doesn't seem to have been changed...did you push your changes?